### ISSUE TITLE
Namespace question

### ISSUE THREAD

**Issue by devdattakulkarni** [datastax/cass-operator#135]
The README shows an example of deploying DSE in a namespace and also creating the datacenter instance in the same namespace.

It will be good to address in the README if it is possible to deploy the Operator in one namespace and allow creating clusters in other namespaces. Check this for details:

https://github.com/cloud-ark/kubeplus/blob/master/Guidelines.md#make-custom-controllers-namespace-aware

**Comment by bradfordcp** [comment]
It is definitely possible to run the operator in a separate namespace from the one that it is watching for Custom Resources. I agree it should be clearer how to configure this.

**Comment by jimdickinson** [comment]
Yeah, another issue opened recently also pointed out that we could have a flow where we create a ClusterRole/ClusterRoleBinding to facilitate an operator that can manage resources in other/all namespaces. We'll add something to the helm chart and the docs.
